LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2018 Homely.com.au

```

Comment:
2020? 😉

Reply:
Nope. this was when the license _started_.

(and no, you don't need to do <start>-<current or end>)

Reply:
ref: https://opensource.stackexchange.com/questions/1522/what-should-be-written-in-mit-license-year-full-name